### ISSUE TITLE
Module and method needs

### ISSUE THREAD

**Issue by ptresson** [umr-amap/amap-rs#2]
It would be good to know which helper functions, classes and modules we need. From my own perspective I can already see some functions that I will quickly add up (focussing on torchgeo and deep learning for now) :

- sort black tiles
- normalize data properly
- paste the appropriate number of channels into a pre-trained model
- vizualisation of backbone outputs (e.g. plot UMAP or PCA projection)
